i18n/en.jsonp.js

```diff
@@ -19,8 +19,14 @@ document.localeJson = {
       "configure-title": "Configuration",
       "configure-about-usage-link": "About (Usage)",
       "configure-btn-library": "Browse ZIM Library",
+      "configure-btn-folderselect": "Select Folder",
+      "configure-btn-rescan": "Rescan",
+      "configure-about-rescan-tip": "Rescans your SD Cards and internal memory",
+      "configure-select-file-numbers": "archive(s) found in selected location. ",
       "configure-download-instructions": "This application needs a ZIM archive to work.<br />For full instructions, please see the section",
       "configure-select-instructions": "Please select or drag and drop a .zim file (or all the .zimaa, .zimab etc in case of a split ZIM file):",
+      "configure-select-file-instructions": "Please select the archive you want to use :",
```

Comment:
Eliminate space between "use" and colon. That's French style, but not used in English.